packages/python-jsonschema/python-jsonschema.spec

```diff
@@ -5,13 +5,14 @@
 %global pypi_name jsonschema
 
 Name:           %{?scl_prefix}python-%{pypi_name}
-Version:        3.2.0
-Release:        8%{?dist}
+Version:        4.6.0
+Release:        1%{?dist}
 Summary:        An implementation of JSON Schema validation for Python
 
 License:        MIT
 URL:            https://github.com/Julian/jsonschema
 Source0:        https://files.pythonhosted.org/packages/source/j/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
+Source1:        001-SETUP-CFG
```

Comment:
It might be nice to have a comment above this line stating why it exists (i.e. jsonschema switching to strictly pyobject.toml, this is the setup.cfg from 4.5.1, it's to keep builds working unitl our infrastructure can handle pyboject.toml based building)